java/2.0/modules/ROOT/pages/java-module-upgrade-migrate.adoc

```diff
@@ -0,0 +1,59 @@
+= Upgrading and Migrating Java Module to Version 2.0 
+
+Upgrade Java Module to version 2.x.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|1.x |2.x
+|===
+
+== Changes in This Release
+
+Java Module 2.0 includes these changes.
+
+* Java Module (and its dependencies) are now compilable and compatible with only Java 17. 
```

Comment:
* Java Module (and its dependencies) are now compilable and compatible only with Java 17.